api/server.js

```diff
@@ -1,5 +1,7 @@
 const app = require("./app");
+const cron = require('./cron-jobs/run-cron-jobs')
```

Comment:
Tiny thing, but semi-colon still needed here 🙂

Reply:
Done. Really this time.